src/openapi/streaming/subscription.js

```diff
@@ -442,6 +455,21 @@ Subscription.prototype.removeStateChangedCallback = function(callback) {
     }
 };
 
+Subscription.prototype.processUpdate = function(message, type) {
+    message.Data = this.serializer.parse(message.Data, this.SchemaName);
+    this.onUpdate(message, type);
+};
+
+Subscription.prototype.processSnapshot = function(response) {
+    if (response.Schema && response.SchemaName) {
+        this.SchemaName = response.SchemaName;
+        this.serializer.addSchema(response.Schema, response.SchemaName);
+    }
+
+    // Snapshot protobuf serialization is currently 'nice to have' on openapi side. So, for now, always using JSON.

```

Comment:
more professional comment maybe?
// Snapshot protobuf serialization is currently not supported
?